funnel/models/project.py

```diff
@@ -87,6 +87,9 @@ class Project(BaseScopedNameMixin, db.Model):
     inherit_sections = db.Column(db.Boolean, default=True, nullable=False)
     labels = db.Column(JsonDict, nullable=False, server_default='{}')
 
+    featured_sessions = db.relationship("Session",
+        primaryjoin="and_(Session.project_id==Project.id, Session.featured==True)", back_populates="project")
```

Comment:
Should be `and_(Session.project_id == id, Session.featured == True)`

Reply:
@jace using only `id` instead of `Project.id` throws error - 
> ArgumentError: Could not locate any relevant foreign key columns for primary join condition 'session.project_id = :project_id_1 AND session.featured = true' on relationship Project.featured_sessions. Ensure that referencing columns are associated with a ForeignKey or ForeignKeyConstraint, or are annotated in the join condition with the foreign() annotation.

Reply:
If it points to a column in the model, it'll return featured sessions from other projects. Test and confirm.

Reply:
@jace tested across 2 projects. Each one shows the respective featured sessions. Not from each other.

Reply:
I took the usage reference from here (the example) https://docs.sqlalchemy.org/en/latest/orm/join_conditions.html#specifying-alternate-join-conditions

Reply:
Okay, this is good then. I guess SQLAlchemy is being intelligent about value substitution.